### ISSUE TITLE
What should we do if user change table replica identity when using debezium as a CDC for PostgreSQL

### ISSUE THREAD

**Issue by chenzl25** [risingwavelabs/risingwave#5892]
### Describe the bug

When we use debezium as a CDC to capture the change of PostgreSQL, we need to use statement `ALTER TABLE t1 REPLICA IDENTITY FULL` to ensure the `before` field of the update event is filled. What happen when at first the user don't specify replica identity, but later they change it to `FULL`. I found that we will skip all the update event if the before field is missing. After the table replica identity is changed to `FULL`, the sanity check might fail. So in production scenario, what is the best practice?

**Comment by hzxa21** [comment]
Correct me if i am wrong, there is a bootstrap phase on setting up CDC, during which a snapshot of the upstream db is read. We can ingest the CDC stream only after finishing bootstraping. That being said, i think we can start over and re-bootstrap after ALTER.

**Comment by chenzl25** [comment]
> Correct me if i am wrong, there is a bootstrap phase on setting up CDC, during which a snapshot of the upstream db is read. We can ingest the CDC stream only after finishing bootstraping. That being said, i think we can start over and re-bootstrap after ALTER.

Yes, you are right. What I am afraid of is user intended or accidental ALTER causes risingwave panics.

**Comment by tabVersion** [comment]
> > Correct me if i am wrong, there is a bootstrap phase on setting up CDC, during which a snapshot of the upstream db is read. We can ingest the CDC stream only after finishing bootstraping. That being said, i think we can start over and re-bootstrap after ALTER.
> 
> Yes, you are right. What I am afraid of is user intended or accidental ALTER causes risingwave panics.

In current impl, source will ignore unsupported messages and send a warning. So we are ignoring the ALTER and trying our best to parse later messages with schema unchanged

**Comment by chenzl25** [comment]
This behavior may cause our state table sanity check failure. However, in risingwave release mode, we will disable the sanity check. I am thinking about any streaming operators required strict sanity check. cc @TennyZhuang
